corehq/apps/hqmedia/templates/hqmedia/audio_translator.html

```diff
@@ -0,0 +1,60 @@
+{% extends "hqmedia/uploader_base.html" %}
+{% load compress %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% block js %}{{ block.super }}
+  {% compress js %}
+    <script src="{% static 'hqwebapp/js/widgets.js' %}"></script>
+  {% endcompress js %}
+{% endblock js %}
+
+{% block page_content %}
+  <p class="lead">{{ current_page.page_name }}</p>
+
+  {% url "app_settings" domain app.get_id as app_settings_url %}
+  {% blocktrans %}
+    <p class="help-block">
+      This page generates files to assist in working with audio translation vendors. It generates a zip file that
+      contains two Excel files:
+    </p>
+    <p class="help-block">
+      The first file is for the translator's use. It contains one row for each audio file to create, with columns
+      for the text and the desired filename. This file will contain a row for each question that has an audio filename
+      specified but has not yet had an audio file uploaded. If multiple questions use have identical text, the file
+      will contain only one row for that set of questions. It will arbitrarily select one of the questions' filenames.
+    </p>
+    <p class="help-block">
+      The second file is for updating audio paths for questions with duplicate text. This file will contain a row for
+      each question that should have its audio path changed due to the de-duplication described above. After downloading
+      and sanity checking, re-upload this file in <a href="{{ app_settings_url }} " target="_blank">Bulk Application Translations</a>.
+    </p>
+  {% endblocktrans %}
+
+  <form class="form-horizontal" method="get">
```

Comment:
why keeping this as "get"?

Reply:
Are you saying it should be a post? It doesn't modify anything on the server.